src/mscorlib/shared/Interop/Unix/System.Native/Interop.GetSystemTimeAsTicks.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetSystemTimeAsTicks")]
```

Comment:
It would be useful to measure how much overhead the PInvoke adds on Unix. I suspect we may need to use the PInvoke on Unix as well to avoid the perf regression there.